cinema_service/settings.py

```diff
@@ -137,3 +137,6 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_URL = "/media/"
```

Comment:
ignore media files when you push something to repo. so let's remove the files you pushed and ignore them